tests/dialects/test_postgres.py

```diff
@@ -775,7 +775,7 @@ def test_postgres(self):
             },
             write={
                 "sqlite": "1 / CAST(CAST(CAST(4 AS REAL) / 2 AS INTEGER) AS REAL)",
-                "duckdb": "1 / CAST(4 // 2 AS DECIMAL)",
+                "duckdb": "1 / CASE WHEN CAST(4 // 2 AS DECIMAL) = 0 THEN ERROR('Division by zero error') ELSE CAST(4 // 2 AS DECIMAL) END",
```

Comment:
Doing an integer division by zero isn't handled yet. We may need to deal with this separately, because I noticed the semantics aren't necessarily the same as normal division...

ClickHouse:

```
:) select 1 / 0;
   ┌─divide(1, 0)─┐
1. │          inf │
   └──────────────┘

:) SELECT 1 // 0;
   ┌─1─┐
1. │ 1 │
   └───┘
```

DuckDB:

```
D SELECT 1 / 0;
┌─────────┐
│ (1 / 0) │
│ double  │
├─────────┤
│   inf   │
└─────────┘
D SELECT 1 // 0;
┌──────────┐
│ (1 // 0) │
│  int32   │
├──────────┤
│   NULL   │
└──────────┘
```

OTOH, Postgres is consistent:

```
georgesittas=# SELECT 1 / 0;
ERROR:  division by zero
georgesittas=# SELECT DIV(1, 0);
ERROR:  division by zero
```